packages/vscode-extension/src/devices/IosSimulatorDevice.ts

```diff
@@ -395,3 +412,18 @@ function convertToSimctlSize(size: DeviceSettings["contentSize"]): string {
       return "extra-extra-extra-large";
   }
 }
+
+function convertToSimctlPermission(permissionType: AppPermissionType): string {
```

Comment:
Do we expect more to come? Right this function is doing nothing as far as I can see.

Reply:
I updated it such that I have a separate list of accepted service names. The idea was that we levarage typescript to catch the case when we add new item in the future and it isn't mapping directly to the name that we can use with simctl